inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/TextFileSource.java

```diff
@@ -76,6 +84,28 @@ public List<Reader> split(JobProfile jobConf) {
         return result;
     }
 
+    @Override
+    public void initSourceMeta(JobProfile jobProfile) {
+        this.sourceMetas = MetaDataUtils.getSourceMeta(jobProfile);
+    }
+
+    private int getSeekPosition(JobProfile jobConf, File file) {
+        int seekPosition;
+        if (jobConf.hasKey(JobConstants.JOB_FILE_COLLECT_TYPE) && FileConstants.INCREMENT.name()
```

Comment:
What about `FULL` mode?

Reply:
Filter out unsupported collect type to reduce task submission

Reply:
Please, to see the code.
![image](https://user-images.githubusercontent.com/17596132/182343385-ea8fa868-ff4f-4c1a-8936-1c98e7bddccd.png)